src/css/main.css

```diff
@@ -249,6 +249,7 @@ body {
 #selected-courses-bar {
   overflow-y: auto;
   justify-content: space-between;
+  min-height: 38px;
```

Comment:
Hmm, hard-coding pixel sizes is not really great CSS--it becomes inconsistent depending on available fonts, different devices/DPIs, etc.  Have you tried `min-content` or `max-content` instead, since the goal is to fit the container to its content?